indigo_app/static/javascript/indigo/views/document_editor.js

```diff
@@ -281,11 +281,11 @@
 
       var self = this,
           renderCoverpage = this.parent.fragment.parentElement === null,
-          $akn = this.$('.document-workspace-content .akoma-ntoso'),
+          $akn = this.$('.document-workspace-content la-akoma-ntoso'),
           coverpage;
 
       // reset class name to ensure only one country class
-      $akn[0].className = "akoma-ntoso spinner-when-empty country-" + this.parent.model.get('country');
+      $akn[0].className = "akoma-ntoso spinner-when-empty";
```

Comment:
We should drop the `akoma-ntoso` class since it's not longer applying indigo-web styling? If there are custom styles bound to it indigo, can you update them to la-akoma-ntoso?

Reply:
done. targeted la-akoma-ntoso for places i considered to be custom stylings of indigo web